stanford/roles/deployment/tasks/main.yml

```diff
@@ -27,27 +30,27 @@
   #   Name: "{{ COMMON_DEPLOYMENT }}-gateway-internet"
 - debug: var=deployment_gateway
 
-- name: Create NAT subnet
+- name: Create public subnet
```

Comment:
From here to the end of the file, use `public` instead of `nat` and pull values from `DEPLOYMENT_` instead of `CLUSTER_`.